frontend/__tests__/src/hooks/useVerifyPipelineToolEffect.test.tsx

```diff
@@ -11,24 +11,23 @@ describe('use verify pipelineTool state', () => {
 
     expect(result.current.isLoading).toEqual(false);
   });
-  it('should set error message when get verify pipelineTool throw error', async () => {
-    jest.useFakeTimers();
+
+  it('should set error message when get verify pipeline response status 500', async () => {
```

Comment:
should set error message when verifying pipeline given response status 500

Reply:
ok